### DISCUSSION TITLE
[ AUDIO ] ogg loops with vgstream not working well. (CANARY)

### DISCUSSION THREAD

**Discussion by TurBoss** [julianxhokaxhiu/FFNx#262] [Q&A] [answered]
some song loops fail to load with label "LOOP_START" , "LOOP_END" and "LOOP_LENGTH"

some other song loop fine

**To Reproduce**
create a looped ogg with audacity

1. set "LOOP_START"

**Expected behavior**
song playing in loops forever

**Additional context**
vg stream from ffnx lastest from 7heaven canary channel

Thanks

**Comment by julianxhokaxhiu** [comment, marked as answer]
Hey, I doubt it has an issue as the code is [upstream in VGMStream](https://github.com/vgmstream/vgmstream/blob/becac80999af1a75bb0d8710d6f4c29918bce97a/src/meta/ffmpeg.c#L178) and there are other mods using this feature currently.
Please consider using `LoopStart`, `LoopEnd` and `LoopLength` instead.

Regarding the part of "it loops forever" that's what is meant to happen. As soon as you set how to loop through one of either Start or End tags, the song will loop forever.

Thank you!

**Reply by TurBoss** [reply]
ok found that my friend has a brick as computer and only fails for he

Thanks!

**Reply by julianxhokaxhiu** [reply]
Welcome :)
